agent/backend/pktvisor/scrape.go

```diff
@@ -156,9 +157,78 @@ func (p *pktvisorBackend) scrapeDefault() error {
 	return nil
 }
 
+func (p *pktvisorBackend) scrapeOtlp() {
+	policyID := "__all"
+	exeCtx, execCancelF := context.WithCancel(p.ctx)
+	exeCtx = context.WithValue(exeCtx, "policy_name", policyID)
```

Comment:
shouldn't this be a name, not ID? also elsewhere it looks like the policy name is just "policy", not "policy_name" so let's make sure we are consistent

Reply:
i change this. but kept policy_name to keep compatibility with current otel implementation. once we get this fully tested we can remove current otel code and rename it